apps/snitch_core/test/support/factory/stock.ex

```diff
@@ -31,6 +31,16 @@ defmodule Snitch.StockFactory do
           stock_location: build(:stock_location)
         }
       end
+
+      def stock_movement_factory do
+        %StockMovement{
+          quantity: 1,
+          action: "",
```

Comment:
Please fill these fields with some example text.

Reply:
not checked the logic behind it. Will add it later